config.toml

```diff
@@ -42,17 +42,22 @@ header_wrapper_class = "header-default-bg-img"
   plainIDAnchors = true
   hrefTargetBlank = true
 
+[[menu.main]]
```

Comment:
This shouldn't be added to the main menu of the site, but instead, be listed in the readme of the repository.